R/Google.R

```diff
@@ -0,0 +1,191 @@
+#' R6 Class containing specific attributes and methods for Google data
+#'
+#' @description Google specific information for downloading, cleaning
```

Comment:
This is quite a complex class it would be good to explain in more detail which data sets are important and what you are going to do with them.